### ISSUE TITLE
Buttons

### ISSUE THREAD

**Issue by gehalber** [everythingtype/brooklynschoolhouse#2]
Would love to apply some of these rules to the button interaction of the site:
http://www.google.com/design/spec/components/buttons.html#buttons-floating-action-button

**Comment by halfempty** [comment]
Yes. Once the elements are in, I'll go back and do a pass on all the buttons to make them match this style.

**Comment by halfempty** [comment]
@gehalber I found a CSS-generated fab in the wild on Google Docs: 

![image](https://cloud.githubusercontent.com/assets/775543/6550093/e5713a70-c5ed-11e4-88a6-eb0dbeb692bb.png)

I've copied the CSS settings for the shadow and hover animation. Here:

http://bksh.etc-nyc.com/

Two elements of the Docs FAB I didn't use. The first is a slight inner glow / bevel which isn't in the  spec. The second is a slight color change (lightening) on hover. Let me know if you'd like to see either of those, or any other changes to the shadow or hover effect.
